src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsServiceTest.kt

```diff
@@ -57,21 +51,6 @@ class CourtCaseEventsServiceTest {
     verifyNoMoreInteractions(telemetryService)
   }
 
-  @Test
-  fun `should call telemetry service when PNC is empty`() { // failing test to show double counting of empty string pnc
-    // Given
-    val person = Person(familyName = "Jones", otherIdentifiers = OtherIdentifiers(pncIdentifier = null))
-
-    // When
-    courtCaseEventsService.processPersonFromCourtCaseEvent(person)
-
-    // Then
-    verify(personRecordService, never()).createNewPersonAndDefendant(person)
-    verify(telemetryService).trackEvent(TelemetryEventType.MISSING_PNC, emptyMap())
-    verify(telemetryService, never()).trackEvent(INVALID_PNC, mapOf("PNC" to ""))
-    verifyNoMoreInteractions(telemetryService)
-  }
-
```

Comment:
This is covered here:

https://github.com/ministryofjustice/hmpps-person-record/blob/9b6ebaba994608a1e47a9fe3dcb396d22d65d08c/src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt#L88